tests/static-analysis/assert-notFalse.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Webmozart\Assert\Tests\StaticAnalysis;
+
+use Webmozart\Assert\Assert;
+
+/**
+ * @param mixed $value
```

Comment:
The reason this fails is because asserting that something is not false does not guarantee it is a bool.

You could do something like this:

```php
/**
 * @param false|string $value
 */
function consume($value): string
{
    Assert::notFalse($value);

    return $value;
}
```

Reply:
That makes a lot of sense. Changed it.